config/llmisvc/config-llm-prefill-worker-data-parallel.yaml

```
@@ -0,0 +1,124 @@
+      volumes:
+        - emptyDir: { }
+          name: home
+        - emptyDir:
+            medium: Memory
+            sizeLimit: 1Gi
+          name: dshm
+        - emptyDir: { }
+          name: model-cache
+        - name: tls-certs
+          secret:
+            secretName: "{{ ChildName .ObjectMeta.Name `-kserve-self-signed-certs` }}"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)